operator/pkg/helm/helm.go

```diff
@@ -63,7 +63,7 @@ type TemplateRenderer interface {
 // The format of helmBaseDir and profile strings determines the type of helm renderer returned (compiled-in, file,
 // HTTP etc.)
 func NewHelmRenderer(operatorDataDir, helmSubdir, componentName, namespace string) TemplateRenderer {
-	dir := filepath.Join(ChartsSubdirName, helmSubdir)
+	dir := strings.Join([]string{ChartsSubdirName, helmSubdir}, "/")
```

Comment:
not a criticism of your code but of embedd - this kind of sucks? :slightly_smiling_face:. You would think that you would be able to write code for both embed and standard files

Reply:
Actually - does this work? since I think we do use this code for embedded and standard files?

This may fix windows for `istioctl install` but not `istioctl install -d manifests/` possibly?

Reply:
I haven't tried anything with passed in paths, and I suspect that may cause more issues.  I at least wanted to publicize what changes were needed to make a few of the failing commands I was trying to work. Things like `istioctl manifest generate --dry-run --set profile=demo` and  `istioctl manifest generate --dry-run --set profile=demo --manifests manifests` which had different errors originally now both yield generated output so things are getting further.

Reply:
Comment below, but I did try using --manifests(-d) and --file with `istioctl manifest` and that worked. I've asked for other things to test in Slack. For now, I'm not seeing any failures, so I'm not pursuing more changes and this is ready to go.